Resources/Private/CustomStylingForCkEditor/src/manifest.js

```diff
@@ -1,25 +1,28 @@
 import manifest from '@neos-project/neos-ui-extensibility';
-import {IconButton} from '@neos-project/react-ui-components';
+import {$add, $get} from 'plow-js';
+import ExamplePlugin from './examplePlugin';
+import ExampleButton from './ExampleButton';
 
-manifest('main', {}, globalRegistry => {
-    const ckEditorRegistry = globalRegistry.get('ckEditor');
+const addPlugin = (Plugin, isEnabled) => (ckEditorConfiguration, options) => {
```

Comment:
Would be nice to expose this helper function via extensibility api, so it wouldn't be necessary to copy-paste it all the time. But can be done later ofc.